Fixes for second round of peer reviews

asmeurer opened the pull request:
See https://github.com/sympy/sympy-paper/issues/215

Everyone push to this branch.

asmeurer committed:
Fix A = B citation (editor note 5)

The correct citation was already in the bib file.

Item of note: for books, always get the citation from Google Books, not Google
Scholar.

asmeurer committed:
Update make diff to make the latest diff

asmeurer committed:
Indicate that oo is two "o"s (reviewer note 2)

asmeurer committed:
Editor note 3

asmeurer committed:
Editor note 4

asmeurer committed:
Editor note 6

asmeurer committed:
Editor note 7

asmeurer committed:
Start the second rebuttal

asmeurer committed:
Skip the second coverletter and rebuttal when building the diff

asmeurer committed:
Start rebuttal2 with the editor comments

asmeurer committed:
Fix unicode quote

asmeurer committed:
Add in reviewer comments to the rebuttal2

asmeurer committed:
Fix LaTeX error in rebuttal2.tex

asmeurer committed:
Fix math

asmeurer committed:
Use "integrate tan(x)" for SymPy Gamma link (reviewer 3 note 6)

asmeurer committed:
Address reviewer 3 point 5

asmeurer committed:
Fix reviewer 3 point 4.

asmeurer committed:
Fix reviewer 3 points 2 and 3

certik committed:
Fix Editor comment 1.

asmeurer commented on assumptions.tex:
```diff
@@ -1,9 +1,11 @@
-such identity holds.
+unless the assumptions allow them. For instance, the simplification $\sqrt{t^2}
+= t$ holds if $t$ is nonnegative ($t\ge 0$). However, for a complex $t$, the
+simplification is in general invalid, the correct identity that holds for all
```
I would either remove the last part of the sentence ("the correct identity...") or move it to a footnote.

certik replied:
I moved it into a footnote in 17e83546e385b01559ccdff374fd431a4209eca8.

I would like it to be there, since the editor raised a question about it, so instead of "talk", let's provide an actual identity for `sqrt(x^2)` in terms of `x` that actually works for all complex `x`.

asmeurer replied:
That's fine. Although as far as I know SymPy doesn't implement this formula (I could be wrong about that, though).

asmeurer replied:
Also, why not use the definition using `exp`. Isn't that more standard than the one you have?

certik replied:
You can write it using `exp`, but that's even more complicated, I just simplified `exp(pi*I)=-1`. See http://www.theoretical-physics.net/dev/math/complex.html#examples

asmeurer replied:
Can you update the rebuttal for this as well?

certik replied:
Done in 66accab3dbc69699e518657163885e5a114e8023.

certik committed:
Move it into a footnote

certik committed:
Clarify the branch cut

certik committed:
Update rebuttal2

certik committed:
Fix a typo

asmeurer committed:
Update make clean

asmeurer committed:
Don't remove authors.md with make clean

asmeurer committed:
Fix new files in the diff script

asmeurer committed:
Add periods at the end of all the figure captions

asmeurer committed:
Fix garbled text

asmeurer committed:
Remove unused live.tex

asmeurer committed:
Grammar fix

asmeurer committed:
Fix ~ before a \ref

asmeurer committed:
Add a paragraph to the main paper about the supplement

Fixes reviewer 3 note 1.

asmeurer committed:
Make sure the supplement is built before the paper

Otherwise the cross-document references won't work correctly.

asmeurer committed:
Reference specific section numbers for the supplement throughout the paper

Since we are doing cross-document linking for the intro section anyway, we
might as well do this.

asmeurer committed:
Note that we added the section references in the rebuttal2

asmeurer committed:
Write the coverletter2

asmeurer commented:
OK, this is now ready to be reviewed and signed-off by people. @certik do you want to handle signoffs (including for the previous changes, for people who haven't yet)? I would like to resubmit this by the end of the week.

asmeurer commented:
Actually we still need to wait from PeerJ to see if there are any technical changes that need to be made. So the end of this week may be too soon. The deadline is 40 days (from October 17). The changes are basically done, so obviously I'd like to submit it well before that. 

One thing I'm unclear about: should we remove the line numbers when we resubmit, since the decision this time was "minor changes"?

asmeurer committed:
Move a paragraph up

This gets rid of a block of vertical whitespace before it, because the table
is there.

asmeurer committed:
Only build the paper and supplement in diff.xsh

That should make it faster

scopatz commented:
I am happy with it in its current state (9ca7e50), and sign off on it. Once again, great work all and @asmeurer!

certik commented:
@asmeurer is it possible to create a pdf document that people can sign off? Currently some stuff is metadata, and it's really confusing what will end up in the final pdf as created by PeerJ.

If not, I would like to see the final pdfs, as well as metadata, that you submitted to the journal. So that people know exactly what they are signing off. Can you help me with this?

Once I have the documents, yes, I will organize it.

asmeurer commented:
@certik you can use https://github.com/isuruf-bot/sympy-paper/commit/b3b611e473225d6144876c6556e3b64a4fa36347

certik commented:
@asmeurer thanks. Do you have a link to the exact metadata, that you submitted with it? Acknowledgements, etc.?

asmeurer commented:
Only if you are able to read this: https://peerj.com/manuscripts/11410/declarations/#reqStatement. 

There's actually a lot of forms that I have to fill out. You can see everything from that manuscript html I emailed you on October 6. Was there anything on that page that needed to change? I can send you the latest version if you want.

certik committed:
Reformulate the sqrt(t^2) case

certik committed:
Move the text in parentheses into a footnote

certik committed:
Update rebuttal2.tex

Upabjojr committed:
dependencies for SymPy version 1.0

asmeurer commented:
So I contacted PeerJ, and apparently the technical changes part of their email was a bug. There are no further changes forthcoming. So I guess I'll go ahead and submit what is here.

asmeurer committed:
Update date on the coverletter2

certik commented:
@asmeurer perfect. Keep us updated how it goes.

asmeurer commented:
So I just noticed a small issue. The bottom of the features table is running over into the footnote on page 3. Here is a screenshot. 

<img width="440" alt="screenshot 2016-11-09 21 00 20" src="https://cloud.githubusercontent.com/assets/71486/20161915/89d735f2-a6bf-11e6-8b17-72ed7c8341a0.png">

asmeurer commented:
I have no idea how to fix it.

certik commented:
You can ask the journal to help with formatting like this.

asmeurer commented:
I've submitted it.
